generate_brick.py

```diff
@@ -682,12 +682,15 @@ def add_definitions(graph=G):
     with open("./bricksrc/definitions.csv", encoding="utf-8") as dictionary_file:
         dictionary = csv.reader(dictionary_file)
 
-        # skip the header
-        next(dictionary)
+        header = next(dictionary)
 
         # add definitions, citations to the graph
         for definition in dictionary:
             term = URIRef(definition[0])
+            if len(definition) > len(header):
```

Comment:
It appears some row's definitions are not wrapped in parentheses and using commas within the definition, which is causing parsing issues. This new condition should catch most of them.

Reply:
There was one bad row which you can fix in definitions.csv (I think it is the Tablet row)